src/hyperion/experiment_plans/rotation_scan_plan.py

```diff
@@ -268,11 +266,23 @@ def rotation_scan(composite: RotationScanComposite, parameters: Any) -> MsgGener
     def rotation_scan_plan_with_stage_and_cleanup(
         params: RotationInternalParameters,
     ):
+        motor_time_to_speed = yield from bps.rd(composite.smargon.omega.acceleration)
+        max_vel = (
+            yield from bps.rd(composite.smargon.omega.max_velocity)
+            or DEFAULT_MAX_VELOCITY
```

Comment:
Should: Why do we have a default in here? In case the PV doesn't exist - we have bigger problems. For the unit tests - we should just be setting up the mock device properly

Reply:
I think that's very unlikely but I'm not that bothered